### ISSUE TITLE
Test new PR create views

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode-pull-request-github#5100]
Refs: https://github.com/microsoft/vscode-pull-request-github/issues/4403

- [x] anyOS @connor4312
- [x] anyOS @aiday-mar
- [x] anyOS @karthiknadig

Complexity: 5

Authors: @alexr00, @hbons 

[Create Issue](https://github.com/microsoft/vscode-pull-request-github/issues/new?body=Testing+%235100%0A%0A&assignees=alexr00,hbons)

---

We have update the Create PR webview and the Files changed views. Please do exploratory testing on the "Create" views. 

Setup: 
- Install the pre-release build of the extension
- The new views are enabled by default already in the pre-release version
- Open a folder with a GitHub repository that you don't mind create PRs and pushing branches in
- To open the "Create" views, use the "Create Pull Request" command or the "Commit & Create Pull Request" button in the SCM view.

Specific features to test:
- Setting assignees, reviewers, labels, and a milestone from the create view
- After setting assignees/reviewers/labels/milstones, confirm everything in the webview is tabbable
- We try to guess your "into" branch better: If the branch you branched off from has an remote, then that will be your into branch. Otherwise, it will be the default branch for the repo
- In the branch selector, confirm ‘Change Repository…’ lists the repositories in the network
- The "Create" button remembers your last choice
- The "Files Changed" view shows changes even when your branch hasn't been pushed yet
- The "Files Changed" and "Commits" view update when you change branches or push your "from" branch.
- The controls in the webview are disabled when you open the view as we load the data for the view
- The controls in the webview are disabled after you click the "Create" button and while the PR is being created

**Comment by aiday-mar** [comment]
HI @alexr00, I am currently testing the work. While testing, I had the following thoughts:

1. It seems as though it is currently not possible to create labels (perhaps I have not missed the button to do so). However it is possible to create milestones from the view. Do you think it would be useful to be able to create labels too?
2. After I set the assignee, the reviewer and the label I notice the following:

![Screenshot 2023-07-25 at 09 44 05](https://github.com/microsoft/vscode-pull-request-github/assets/61460952/b837726e-5e63-44e8-b6db-7f993442eabb)

There appears to be little space between the reviewer/assignee name and their circular picture. Perhaps a little padding could make it better. 
3. Concerning the `Change repository...` button, I was wondering what repositories exactly I should expect to see?Currently the only repository shown is the one I am currently on. I was expecting to see the other repositories I host on my account.

![Screenshot 2023-07-25 at 09 47 27](https://github.com/microsoft/vscode-pull-request-github/assets/61460952/f8aa361c-b2d6-49fd-8e1f-50eb456d6fff)

I like the new interface!

**Comment by alexr00** [comment]
> It seems as though it is currently not possible to create labels (perhaps I have not missed the button to do so). However it is possible to create milestones from the view. Do you think it would be useful to be able to create labels too?

Seems like a reasonable feature rerquest!

> There appears to be little space between the reviewer/assignee name and their circular picture. Perhaps a little padding could make it better.

@hbons already has a fix ready and is waiting on me: https://github.com/microsoft/vscode-pull-request-github/pull/5101

> Concerning the Change repository... button, I was wondering what repositories exactly I should expect to see?Currently the only repository shown is the one I am currently on. I was expecting to see the other repositories I host on my account

You will only see repos that you have as git remotes.

**Comment by aiday-mar** [comment]
Hey @alexr00, thanks for the response. I have these other git remote repos on my account. They are however not cloned locally on my computer. From what I understand, I should have seen them in the `Change repository...` view?

![Screenshot 2023-07-25 at 10 39 23](https://github.com/microsoft/vscode-pull-request-github/assets/61460952/ba7480a5-b477-430a-a146-c6e2f6d9d3ca)
